torch/jit/frontend.py

```diff
@@ -289,8 +289,15 @@ def _forward(self):
             # Replace potentially unsupported type annotations by "Any"
             arg.annotation = unused_def.args.args[0].annotation
 
-    return build_def(ctx, fn_def, type_line, def_name, self_name=self_name)
+    # If MonkeyType is installed, get all the consolidated type traces
+    # for the arguments from type_trace_db
+    type_trace_db = torch.jit._script._get_type_trace_db()
+    pdt_arg_types = None
+    if monkeytype_trace:
+        _qualname = get_qualified_name(fn)
```

Comment:
Why putting an underscore in front of variable name? underscore signifies private members of modules, it doesn't mean much for a local temporary variable

https://google.github.io/styleguide/pyguide.html#3162-naming-conventions

Reply:
Same for a few other places like `_type`

Reply:
Changed now.